### PR TITLE
Relicense to Apache 2.0

### PR DESCRIPTION
**Pull request by blag**
# TL;DR: WHAT I NEED FROM ALL OF YOU #

This is a relicensing PR, so I need all of you to review this PR and explicitly approve the changes.

I need ALL of you to approve the changes before I can merge it in.

* [x] @lampwins
* [x] @LindsayHill
* [x] @Kami
* [ ] @whites2
* [x] @enykeev

# Details #

Hi everybody, I'd like to relicense this pack from the MIT license to Apache License 2.0, which is what the rest of the packs on StackStorm Exchange, as well as StackStorm itself, are licensed under.

I realize that this may seem like arguing over semantics, but keeping a consistent license between StackStorm and all StackStorm Exchange packs is very useful for our customers and users, because if all Exchange packs are consistently licensed, then our users can install any Exchange pack without worrying about the licensing.

As it stands now, our users have to check that they will be in compliance with the license of every Exchange pack before they install it, which can be annoying.

We now have an Apache 2.0 license check in our pack CI configuration, which is now causing this pack to fail CI.

Under the Apache license, code contributors grant patent licenses to all users (section 3), and users are explicitly _not_ granted a right to use any trademarks of this project (section 6).

### CODE REVIEW

**Review by LindsayHill — APPROVED** [review]

**Review by Kami — APPROVED** [review]

**Review by lampwins — APPROVED** [review]

**Review by enykeev — APPROVED** [review]
welp. Sorry for that I guess :P

**Comment by nzlosh** [comment]
Closing: @whites2 unresponsive on licence change which is a hard blocker on this merge.
